data/baby_saison.json

```diff
@@ -0,0 +1,132 @@
+{
+  "name": "Baby Saison",
```

Comment:
Could you add the B Sides prefix in at the end maybe e.g "Baby Saison - B-Sides"